condacolab.py

```diff
@@ -198,13 +202,25 @@ def install_from_url(
 
     env = env or {}
     bin_path = f"{prefix}/bin"
+    pre_conda_contents = ""
+
+    if env:
+        pre_conda_contents = "".join(f'export {key}={value}\n' for key, value in env.items())
```

Comment:
```suggestion
        pre_conda_contents = "".join(f'export {key}="{shlex.quote(value)}"\n' for key, value in env.items())
```

Remember to import `shlex` in the module.